### ISSUE TITLE
nameparser-0.2.5 install from pypi fails

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gdsaxton/python-nameparser#29]
```
What steps will reproduce the problem?
$ easy_install nameparser==0.2.5

What is the expected output? What do you see instead?
Install fails with the following output:
Searching for nameparser
Reading http://pypi.python.org/simple/nameparser/
Reading http://code.google.com/p/python-nameparser
Best match: nameparser 0.2.5
Downloading 
http://pypi.python.org/packages/source/n/nameparser/nameparser-0.2.5.tar.gz#md5=
e505dd79158cd5114525d2022ba3ab69
Processing nameparser-0.2.5.tar.gz
Writing /tmp/easy_install-MRHi5B/nameparser-0.2.5/setup.cfg
Running nameparser-0.2.5/setup.py -q bdist_egg --dist-dir 
/tmp/easy_install-MRHi5B/nameparser-0.2.5/egg-dist-tmp-6p7utJ
Traceback (most recent call last):
  File "setup.py", line 67, in <module>
    "http://pypi.oreilly.com/package/all_files",
  File "/usr/lib/python2.6/distutils/core.py", line 152, in setup
    dist.run_commands()
  File "/usr/lib/python2.6/distutils/dist.py", line 975, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.6/distutils/dist.py", line 995, in run_command
    cmd_obj.run()
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/command/develop.py", line 27, in run
    self.install_for_development()
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/command/develop.py", line 129, in install_for_development
    self.process_distribution(None, self.dist, not self.no_deps)
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/command/easy_install.py", line 682, in process_distribution
    [requirement], self.local_index, self.easy_install
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/pkg_resources.py", line 586, in resolve
    dist = best[req.key] = env.best_match(req, self, installer)
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/pkg_resources.py", line 831, in best_match
    return self.obtain(req, installer) # try and download/install
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/pkg_resources.py", line 843, in obtain
    return installer(requirement)
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/command/easy_install.py", line 598, in easy_install
    return self.install_item(spec, dist.location, tmpdir, deps)
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/command/easy_install.py", line 628, in install_item
    dists = self.install_eggs(spec, download, tmpdir)
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/command/easy_install.py", line 823, in install_eggs
    return self.build_and_install(setup_script, setup_base)
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/command/easy_install.py", line 1103, in build_and_install
    self.run_setup(setup_script, setup_base, args)
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/command/easy_install.py", line 1089, in run_setup
    run_setup(setup_script, args)
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/sandbox.py", line 33, in run_setup
    lambda: execfile(
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/sandbox.py", line 81, in run
    return func()
  File "/var/lib/jenkins/jobs/pycdm_service_tests/workspace/.pyenv/lib/python2.6/site-packages/distribute-0.6.34-py2.6.egg/setuptools/sandbox.py", line 35, in <lambda>
    {'__file__':setup_script, '__name__':'__main__'}
  File "setup.py", line 6, in <module>
    here = os.path.abspath(os.path.dirname(__file__))
  File "/tmp/easy_install-MRHi5B/nameparser-0.2.5/nameparser/__init__.py", line 8, in <module>
  File "/tmp/easy_install-MRHi5B/nameparser-0.2.5/nameparser/parser.py", line 8, in <module>
AttributeError: 'module' object has no attribute 'NullHandler'

What version of the product are you using? On what operating system?
0.2.5, on Ubuntu and Mac OSX.


```

Original issue reported on code.google.com by `meisenb...@gmail.com` on 13 Feb 2013 at 1:28

**Comment by GoogleCodeExporter** [comment]
```
Sorry about that. I failed to noticed that python 2.6 doesn't have the null 
handler.

I pushed a change that should fix that and bumped the version number to 0.2.6. 
If you could give it a try and confirm that it works for you I will roll it out 
to pypi.


```

Original comment by `dere...@gmail.com` on 13 Feb 2013 at 3:08
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Confirmed, the new version works for me. Thanks for the quick turnaround!
```

Original comment by `meisenb...@gmail.com` on 13 Feb 2013 at 7:08
